### DISCUSSION TITLE
Hcp and bcc lamellae ti-6al-4v

### DISCUSSION THREAD

**Discussion by p-antonioni** [neperfepx/neper#834] [Q&A] [answered]
Hi!
I am quite new to Neper and I have some questions regarding the generation of a synthetic columnar lamellar microstructure of Ti-6Al-4V.
I am using previous beta grains to derive the alpha and beta lamellae.
Is there a way to create a tessellation that obey to the Burger relation between alpha(hcp) and beta(bcc) phases?
Maybe specifying a distribution or using the normal parameter in -morpho lamellar(w=, v=)
Thanks in advance for help and suggestions.

**Comment by rquey** [comment, marked as answer]
See the new option "[-ori](https://neper.info/doc/neper_t.html#cmdoption-ori) `ti_beta2alpha`" and https://github.com/neperfepx/neper/discussions/863.
